lms/djangoapps/course_home_api/toggles.py

```diff
@@ -6,7 +6,16 @@
 
 WAFFLE_FLAG_NAMESPACE = 'course_home'
 
-COURSE_HOME_MICROFRONTEND_PROGRESS_TAB = CourseWaffleFlag(  # lint-amnesty, pylint: disable=toggle-missing-annotation
+# .. toggle_name: course_home.course_home_mfe_progress_tab
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This toggle controls the user interface behavior of the progress tab in
+#   the Learning Management System. When set to True, the progress tab utilizes the newly introduced
+#   Learning MFE graphs. When set to False (default), it utilizes existing grade graph from edx-platform.
+# .. toggle_use_cases: temporary
```

Comment:
Temporary toggles require `toggle_target_removal_date`. See https://edx.readthedocs.io/projects/edx-toggles/en/latest/how_to/documenting_new_feature_toggles.html#toggle-target-removal-date. I'd just use something like `2024-01-01`, or anything like that. We can always push these out, but it can be a DEPR reminder.